crates/engine/schema/src/builder/graph/directives/resolvers.rs

```diff
@@ -79,13 +86,23 @@ fn create_extension_resolvers(ingester: &mut DirectivesIngester<'_, '_>) -> Resu
                 }
                 ExtensionDirectiveType::Resolver => {
```

Comment:
You add a test for `field_resolver` (legacy), but you would need to add a another for `resolver` case. Can do it afterwards if you want